base_report_to_printer/report.py

```diff
@@ -44,11 +44,12 @@ def print_document(self, cr, uid, ids, report_name, html=None,
             )
         return printer.print_document(report, document, report.report_type)
 
-    @api.v8
+    @api.v8  # noqa
```

Comment:
Yeah, I see. I didn't remember this one. Well, I think it's needed, because there's no other option in this case.